entt/include

```diff
@@ -0,0 +1 @@
+../upstream/src
```

Comment:
You also need to add the `.gittattributes` with the line stating that symlink is a directory for windows support, see last paragraph of section 2 there: https://build2.org/article/symlinks.xhtml